mathesar_ui/src/component-library/index.ts

```diff
@@ -5,38 +5,43 @@ export { default as EventHandler } from './common/utils/EventHandler';
 export * from './common/utils/formatUtils';
+export * from './common/utils/ImmutableSet';
 export * from './common/utils/pauseableTweened';
 
 // Actions
-export { default as portal } from './common/actions/portal';
-export { default as popper } from './common/actions/popper';
 export { default as clickOffBounds } from './common/actions/clickOffBounds';
+export { default as popper } from './common/actions/popper';
+export { default as portal } from './common/actions/portal';
 
 // Simple Components
-export { default as TextAvatar } from './text-avatar/TextAvatar.svelte';
-export { default as TextInput } from './text-input/TextInput.svelte';
-export { default as TextArea } from './text-area/TextArea.svelte';
-export { default as Checkbox } from './checkbox/Checkbox.svelte';
 export { default as Button } from './button/Button.svelte';
+export { default as CancelOrProceedButtonPair } from './cancel-or-proceed-button-pair/CancelOrProceedButtonPair.svelte';
+export { default as Checkbox } from './checkbox/Checkbox.svelte';
+export { default as CheckboxGroup } from './checkbox-group/CheckboxGroup.svelte';
 export { default as Icon } from './icon/Icon.svelte';
+export { default as Label } from './label/Label.svelte';
+export { default as LabeledInput } from './labeled-input/LabeledInput.svelte';
 export { default as Progress } from './progress/Progress.svelte';
-export { default as CancelOrProceedButtonPair } from './cancel-or-proceed-button-pair/CancelOrProceedButtonPair.svelte';
-export { default as Skeleton } from './skeleton/Skeleton.svelte';
+export { default as Radio } from './radio/Radio.svelte';
 export { default as RadioGroup } from './radio-group/RadioGroup.svelte';
 export { default as Seesaw } from './seesaw/Seesaw.svelte';
+export { default as Skeleton } from './skeleton/Skeleton.svelte';
 export { default as Spinner } from './spinner/Spinner.svelte';
 export { default as SpinnerButton } from './spinner-button/SpinnerButton.svelte';
+export { default as TextArea } from './text-area/TextArea.svelte';
+export { default as TextAvatar } from './text-avatar/TextAvatar.svelte';
+export { default as TextInput } from './text-input/TextInput.svelte';
 
 // Compound Components (Ordered)
+export { default as Dropdown } from './dropdown/Dropdown.svelte';
+export { default as FileUpload } from './file-upload/FileUpload.svelte';
 export { default as Notification } from './notification/Notification.svelte';
-export { default as TabContainer } from './tabs/TabContainer.svelte';
-export { default as Tree } from './tree/Tree.svelte';
 export { default as Pagination } from './pagination/Pagination.svelte';
-export { default as Dropdown } from './dropdown/Dropdown.svelte';
 export { default as Select } from './select/Select.svelte';
-export { default as FileUpload } from './file-upload/FileUpload.svelte';
+export { default as TabContainer } from './tabs/TabContainer.svelte';
+export { default as Tree } from './tree/Tree.svelte';
```

Comment:
Ordering here is maintained based on components relying on one another. For eg., Select relies on Dropdown, so Dropdown would be placed first and Select would be placed after. This would help us easily identify compound components with high level of inter-dependencies.

Do you feel alphabetical ordering would be more advantageous here?